kinto_http/__init__.py

```diff
@@ -259,6 +267,89 @@ def delete_buckets(self, safe=True, if_match=None, if_exists=False):
         resp, _ = self.session.request('delete', endpoint, headers=headers)
         return resp['data']
 
+    # Groups
+
+    def get_groups(self, bucket=None):
+        endpoint = self.get_endpoint('groups', bucket=bucket)
+        return self._paginated(endpoint)
+
+    def create_group(self, group=None, bucket=None,
```

Comment:
If you do a `POST /buckets/x/groups` the server will randomly generate a group id for you.

```
➜  ~ echo '{"data":{"members":[]}}' | http POST https://kinto.dev.mozaws.net/v1/buckets/x/groups --auth user:pass
HTTP/1.1 201 Created
Access-Control-Expose-Headers: Retry-After, Content-Length, Alert, Backoff
Connection: keep-alive
Content-Length: 172
Content-Type: application/json; charset=UTF-8
Date: Mon, 12 Sep 2016 08:07:15 GMT
Server: nginx

{
    "data": {
        "id": "1xEdx9UY", 
        "last_modified": 1473667635578, 
        "members": []
    }, 
...
```